packages/livekit-rtc/package.json

```diff
@@ -56,7 +65,7 @@
   },
   "scripts": {
     "prebuild": "node -p \"'export const SDK_VERSION = ' + JSON.stringify(require('./package.json').version) + ';'\" > src/version.ts",
-    "build:tsc": "pnpm prebuild && tsc && cp -r src/napi dist/",
+    "build:tsc": "pnpm prebuild && tsup --onSuccess \"tsc --declaration --emitDeclarationOnly\" && cp -r src/napi dist/",
```

Comment:
`tsc` is misleading here, not sure what to change it to though.

Reply:
maybe just ts?

Reply:
also: tsup docs suggest to use `tsup-node` to invoke for building node libs

Reply:
that didn't work for me for some reason.